parsers/JP.py

```diff
@@ -39,6 +40,22 @@
 ZONES_ONLY_LIVE = ["JP-TK", "JP-CB", "JP-SK"]
 
 
+def get_wind_capacity(datetime: datetime, zone_key):
+    assert zone_key == "JP-HKD"
+    ZONE_CONFIG = ZONES_CONFIG[zone_key]
+    capacity = ZONE_CONFIG["capacity"]["wind"]
+    if zone_key == "JP-HKD":
+        if datetime.year <= 2019:
+            capacity = 480
+        elif datetime.year == 2020:
+            capacity = 520
+        elif datetime.year >= 2021:
+            capacity = 577
+    else:
+        raise NotImplementedError("This parser can only fetch wind capacity for JP-HKD")
```

Comment:
Won't this assertation make the else statment redundant as it will throw before it is ever reached?

You could also use a ParserException instead to log the zone_key but I don't think that is required.

Reply:
Actually, you are right. 

Now I am testing it for `JP-TH` since it uses the same parser and now we get this as an output. `capacity:None` 

![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/50823290/bf4aa923-b8de-47ec-9bf5-d2871cf43a2d)

Do you think that can mess down the road with capacities? Should I only include the dict 

``` 
{'capacity': {'wind': None},
``` 

for zones that are not JP-HKD?

Reply:
I think it would be better to do something like:
`"capacity": capacities if capacities is not None else {}`

That way we don't contaminate the database with wrong capacity values which I think would affect the app.